Sources/Formworks/FWForm/FWFormViews/FWDismissHeader.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  DismissHFWDismissHeadereader.swift
+//  
+//
+//  Created by Cassia Aparecida Barbosa on 16/11/20.
+//
+
+import UIKit
+final class FWDismissHeader: UITableViewHeaderFooterView {
+	
+	// MARK: Declarations
+	@ManualLayout var button: UIButton
+	@ManualLayout private var title: FWLabel
+	
+	static var identifier: String {
+		return String(describing: self)
+	}
+	
+	// MARK: Init
+	override init(reuseIdentifier: String?) {
+		super.init(reuseIdentifier: reuseIdentifier)
+		setupButtonConstraints()
+		setupTitleConstraints()
+		setHeaderButton()
+	}
+	
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+	
+	
+	// MARK: Setup Image Constraints
+	private func setupButtonConstraints() {
+		self.contentView.addSubview(button)
+		button.setTitleColor(.systemBlue, for: .normal)
+		NSLayoutConstraint.activate([
+			button.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -5),
+			button.widthAnchor.constraint(equalToConstant: 75),
```

Comment:
Be careful with hard coded sizes. A form should look good in every iPhone screen size.